frontend/src/components/smart-contracts/RevokeERC20.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {
+  useSmartContracts,
+  SmartContractActions,
+  SmartContractAddresses,
+} from 'components/smart-contracts/useSmartContracts';
+import { TransactionModal } from 'components/smart-contracts/TransactionModal';
+import { ethers } from 'ethers';
+
+export const RevokeERC20 = ({
```

Comment:
This component should probably not be a component. It's rendering a modal, but it's just called `RevokeERC20`. Either we should rename it to `RevokeERC20Modal` or do this logic elsewhere and just a generic `Modal` component to display content and perform actions.

Reply:
Renamed these to ApproveERC20Modal and RevokeERC20Modal.